pessimistic-transaction.md

```diff
@@ -111,7 +111,11 @@ Pessimistic transactions in TiDB behave similarly to those in MySQL. See the min
 
 2. TiDB does not support `SELECT LOCK IN SHARE MODE`.
 
-    When `SELECT LOCK IN SHARE MODE` is executed, it has the same effect as that without the lock, so the read or write operation of other transactions is not blocked.
+    TiDB does not support `SELECT LOCK IN SHARE MODE` by default.
+
+    You can enable [`tidb_enable_noop_functions`](/system-variables.md#tidb_enable_noop_functions-new-in-v40) to make TiDB compatible with the `SELECT LOCK IN SHARE MODE` statement. The effect of the statement is the same as when there is no lock, and it will not block the read and write operations of other transactions.
+    
+    Starting from v8.3.0, TiDB supports using the [`tidb_enable_shared_lock_upgrade`](/system-variables.md#tidb_enable_shared_lock_upgrade-new-in-v830) system variable to enable the `SELECT LOCK IN SHARE MODE` statement to add locks. However, note that the lock added at this time is not a true shared lock, but consistent with `SELECT FOR UPDATE`, which actually adds an exclusive lock. You can enable this variable when you want to be compatible with the `SELECT LOCK IN SHARE MODE` syntax while blocking writes to prevent data from being modified by parallel write transactions during reads. This variable takes effect regardless of the configuration of [`tidb_enable_noop_functions`](/system-variables.md#tidb_enable_noop_functions-new-in-v40).
```

Comment:
```suggestion
    Starting from v8.3.0, TiDB supports using the [`tidb_enable_shared_lock_promotion`](/system-variables.md#tidb_enable_shared_lock_upgrade-new-in-v830) system variable to enable the `SELECT LOCK IN SHARE MODE` statement to add locks. However, note that the lock added at this time is not a true shared lock, but consistent with `SELECT FOR UPDATE`, which actually adds an exclusive lock. You can enable this variable when you want to be compatible with the `SELECT LOCK IN SHARE MODE` syntax while blocking writes to prevent data from being modified by parallel write transactions during reads. This variable takes effect regardless of the configuration of [`tidb_enable_noop_functions`](/system-variables.md#tidb_enable_noop_functions-new-in-v40).
```